xmtp_mls/migrations/2024-08-07-213816_create-private-preference-store/up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE "private_preferences"(
+    -- The inbox_id the private preferences are owned by
+    "inbox_id" text NOT NULL,
+    -- Enum of the PRIVATE_PREFERENCE_TYPE (GROUP, INBOX, etc..)
+    "valueType" int NOT NULL,
+    -- Enum of PREFERENCE_STATE (ALLOWED, DENIED, etc..)
+    "state" int NOT NULL,
+    -- The value of what has a preference
+    "value" text NOT NULL,
```

Comment:
What do we think about calling it the "entity" just for clarity. I could read value to mean either "the thing that has a preference" or "the preference itself".